Makefile

```diff
@@ -0,0 +1,40 @@
+# stop npm installing for every locally run command but still being a dependency before they can be run first
+INSTALL_STAMP = .npm_installed
+# docker build as dependency but only build it the first itme
+IMAGE_STAMP = .image_built
+
+# headless run
+test: install
+	npx playwright test
+
+# run with browser open
+debug: install
+	npx playwright test --debug
+
+install: $(INSTALL_STAMP)
+
+$(INSTALL_STAMP): package.json package-lock.json
+	npm install
+	npx playwright install
+	npm install --save-dev prettier husky lint-staged
+	npx husky install
```

Comment:
Is this not installed already as it is found in the `package.json` file?

Reply:
Same with playwright

Reply:
made changes in upcoming commit